src/zenml/entrypoints/step_entrypoint_configuration.py

```diff
@@ -354,12 +355,12 @@ def _resolve_class(class_: Type[Any]) -> str:
             json_format.MessageToJson(pb2_pipeline),
             f"--{MAIN_MODULE_SOURCE_OPTION}",
             main_module_source,
-            f"--{STEP_SOURCE_OPTION}",
-            step_source,
-            f"--{INPUT_SPEC_OPTION}",
-            json.dumps(input_spec),
-            f"--{MATERIALIZERS_OPTION}",
-            json.dumps(materializers),
+            f"--{STEP_CLASS_SOURCE_OPTION}",
+            step_class_source,
+            f"--{INPUT_ARTIFACT_SOURCES_OPTION}",
+            json.dumps(input_artifact_sources),
+            f"--{MATERIALIZER_SOURCES_OPTION}",
```

Comment:
just for consistency, why isnt this a MATERIALIZER_CLASS_SOURCES_OPTION?

Reply:
You're absolutely right, mind if I revert `STEP_CLASS_SOURCE` to `STEP_SOURCE` instead? Not sure why I made that change tbh 😄